lib/std/fmt.zig

```diff
@@ -491,6 +491,13 @@ pub fn formatType(
             return format(context, Errors, output, "{}@{x}", .{ @typeName(T), @ptrToInt(value) });
         },
         .Type => return output(context, @typeName(T)),
+        .EnumLiteral => {
```

Comment:
You can simplify this with
```zig
try output(context, ".");
try output(context, @tagName(value));
```

You don't have to recursively call `formatType` when the formatted string is already known.

Reply:
using `formatType` allows for width and (future) alignment options - consider `format("__{s:20}__", .{.hello})`

Reply:
Ah good catch, I didn't pay too much attention to that since many places do not respect the specifiers nor check if they make sense for the type at hand.